src/content/guides/packages.md

```diff
@@ -178,6 +178,78 @@ due to conflicting version constraints in the pubspec.
 To identify out-of-date packages that require editing the pubspec,
 use [`dart pub outdated`][outdated].
 
+## Getting dependencies in production - `pub get --enforce-lockfile`
+
+In some cases `dart pub get` will not give you the exact versions locked in
+`pubspec.lock`. For example:
+
+* If new dependencies are added to `pubspec.yaml` since `pubspec.lock` was created.
+* If the locked version no longer exists in the package repository.
+* If you upgraded the Dart SDK, and some packages no longer are compatible with
+  the sdk.
+  
+In all these cases `dart pub get` will
+
+* Unlock as needed.
+* Notify about any changes in the output.
```

Comment:
It might be helpful to clarify this is talking about package dependency changes, not changes to the textual output.

Does the following make sense?

```suggestion
* Notify you about differences in retrieved packages and the lockfile.
```

Reply:
Tried clarifying